raiden/network/rpc/smartcontract_proxy.py

```diff
@@ -150,7 +151,10 @@ def decode_event(self, log: Dict):
     def encode_function_call(self, function: str, args: List = None):
         return self.get_transaction_data(self.contract.abi, function, args)
 
-    def estimate_gas(self, function: str, *args):
+    def estimate_gas(self, function: str, *args) -> typing.Optional[int]:
+        """Returns a gas estimate for the function with the given arguments or
+        None if the function call will always fail due to Insufficient funds or
```

Comment:
nitpick: The transaction will not "always fail" because of lack of funds, once the account is funded the transaction will succeed. Could you change the wording in here and the logs?